settings/src/components/webauthn.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, Spinner, TextControl } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { Icon, trash } from '@wordpress/icons';
+
+const confirm = window.confirm;
+
+/**
+ * Render the WebAuthn setting.
+ */
+export default function WebAuthn() {
+	const [ step, setStep ] = useState( 'list' );
+
+	// TODO get keys from API.
+	const keys = [
+		{ name: 'Yubikey 5C Nano', id: 1 },
```

Comment:
Could we get this working in this PR so its functionally complete?

Reply:
added in 0a4930a 👍🏻